configs/sites/discover/packages.yaml

```diff
@@ -56,10 +56,10 @@ packages:
     externals:
     - spec: berkeley-db@4.8.30
       prefix: /usr
-  bison:
-    externals:
-    - spec: bison@2.7
-      prefix: /usr
+  #bison:
```

Comment:
Include the `# Don't use, leads to duplicate packages being built` comment warning about duplicate packages?